From 53dc330e9943ad425c6f96cd5803aa49f7732dfc Mon Sep 17 00:00:00 2001
From: Song Xue <quic_songxue@quicinc.com>
Date: Thu, 19 Sep 2024 22:14:29 +0800
Subject: [PATCH 37/63] arm64: dts: qcom: qcs615: Add LLCC support for qcs615

Add LLCC system cache controller entry to support LLCC on qcs615.

Signed-off-by: Song Xue <quic_songxue@quicinc.com>
---
 arch/arm64/boot/dts/qcom/qcs615.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcs615.dtsi b/arch/arm64/boot/dts/qcom/qcs615.dtsi
index 8e1c2f3ceba8..276e873cb134 100644
--- a/arch/arm64/boot/dts/qcom/qcs615.dtsi
+++ b/arch/arm64/boot/dts/qcom/qcs615.dtsi
@@ -589,6 +589,12 @@ qup3_uart0_sleep: qup3-uart0-sleep {
 			};
 		};
 
+		llcc: system-cache-controller@9200000 {
+			compatible = "qcom,qcs615-llcc";
+			reg = <0x0 0x9200000 0x0 0x50000>, <0x0 0x9600000 0x0 0x50000>;
+			reg-names = "llcc0_base", "llcc_broadcast_base";
+		};
+
 		pdc: interrupt-controller@b220000 {
 			compatible = "qcom,qcs615-pdc", "qcom,pdc";
 			reg = <0x0 0xb220000 0x0 0x30000>,
-- 
2.34.1

